docs/testing.rst

```diff
@@ -541,10 +541,10 @@ conditions that hard to reproduce on real server::
    :type app: aiohttp.web.Application
 
    :param writer: object for managing outcoming data
-   :type writer: aiohttp.StreamWriter
+   :type writer: asyncio.StreamWriter
```

Comment:
I'm pretty sure the original type was correct: https://github.com/aio-libs/aiohttp/blob/742a8b6d09b2623670ddede838c913d2a8a4d89e/aiohttp/http_writer.py#L29